deps/wazuh_testing/wazuh_testing/qa_ctl/provisioning/qa_provisioning.py

```diff
@@ -199,5 +199,7 @@ def run(self):
         QAProvisioning.LOGGER.info(f"The instances have been provisioned sucessfully")
 
     def destroy(self):
+        """Destroy all the temporary files created by an instance of this object
+        """
```

Comment:
```suggestion
        """Destroy all the temporary files created by an instance of this object."""
```

Reply:
The new modification can be reviewed in the commit:[bba01140837453f908a9dafefa89308ee9a5103e](https://github.com/wazuh/wazuh-qa/commit/bba01140837453f908a9dafefa89308ee9a5103e)